spec/models/organization_spec.rb

```diff
@@ -76,6 +82,12 @@
             expect(subject.members).to be_empty
           end
         end
+
+        it 'deletes all organization memberships on destroy' do
+          id = subject.id
+          expect { subject.destroy }.
+            to change { OrganizationMembership.where(organization_id: id).count }.to(0)
```

Comment:
Any idea how I can split this line in a nice way? Breaks around `OrganizationMembership.where(...).count`?